### PR TITLE
Csi volume cloning modification

### PR DESCRIPTION
**Pull request by gaoguangze111**
Csi volume cloning page modification

### CODE REVIEW

**Comment by xichengliudui** [comment]
Is it something on the list?
```
https://github.com/k8smeetup/website-tasks/issues
```

**Comment by gaoguangze111** [comment]
Sry I haven't check on the issue list. Do I have to solve an issue for my PR? @xichengliudui

**Comment by chenrui333** [comment]
Will defer to @xichengliudui to proceed with the merge.

File: content/zh/docs/concepts/storage/volume-pvc-datasource.md [hunk 1]
```diff
@@ -52,13 +41,13 @@ This feature requires VolumePVCDataSource feature gate to be enabled:
...
 
-{{< glossary_tooltip text="CSI" term_id="csi" >}} 卷克隆功能增加了在 `dataSource` 字段指定现有的 {{< glossary_tooltip text="PVC" term_id="persistent-volume-claim" >}}s，来表示用户想要克隆的 {{< glossary_tooltip term_id="volume" >}}。
+{{< glossary_tooltip text="CSI" term_id="csi" >}} 卷克隆功能增加了通过在 `dataSource` 字段中指定存在的 {{< glossary_tooltip text="PVC" term_id="persistent-volume-claim" >}}s，来表示用户想要克隆的 {{< glossary_tooltip term_id="volume" >}}。
 
 <!--
 A Clone is defined as a duplicate of an existing Kubernetes Volume that can be consumed as any standard Volume would be.  The only difference is that upon provisioning, rather than creating a "new" empty Volume, the back end device creates an exact duplicate of the specified Volume.
 -->
 
-克隆定义为已有 Kubernetes 卷的副本，可以像任何标准卷一样被使用。唯一的区别就是配置后，后端设备将创建指定卷的精确副本，而不是创建一个“新的”空卷。
+克隆，意思是为已有的 Kubernetes 卷创建副本，它可以像任何其它标准卷一样被使用。唯一的区别就是配置后，后端设备将创建指定完全相同的副本，而不是创建一个“新的”空卷。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/volume-pvc-datasource.md]: > 后端设备将创建指定完全相同的副本

???

**Pull request merged**